lib/shared/permissions.js

```diff
@@ -119,6 +149,37 @@ exports.getEnvironmentCommandPrefix = (environment) => {
 };
 
 /**
+ * @summary Build the elevate command
+ * @function
+ * @private
+ *
+ * @param {String[]} command - command arguments
+ * @param {Object} [environment] - environment variables
+ * @returns {String[]} elevate command
+ *
+ * @example
+ * const command = permissions.buildElevateCommand([ 'foo', 'bar' ], {
+ *   ELECTRON_RUN_AS_NODE: 1
+ * });
```

Comment:
Yeah, I'll update the example. The elevation programs require an array of strings. What this function basically does is transform the command so that it embeds the environment variable definitions.